### DISCUSSION TITLE
Find bin only packages

### DISCUSSION THREAD

**Discussion by exussum12** [composer-unused/composer-unused#92] [Ideas]
## Is your feature request related to a problem? Please describe
Packages used in a CI enviroment use no symbols. For example phpmnd, phpstan etc

## Describe the solution you'd like
If a package provides a "bin" section, check if its used. This can be checked by default with
```grep -ir --exclude-dir=vendor vendor/bin *```
The exclude dir and the binary path should be dynamic (bin-dir defaulting to vendor/bin and vendor-dir being vendor  see https://getcomposer.org/doc/06-config.md#vendor-dir

## Describe alternatives you've considered
These packages should ideally be phar files, but as there is no decent way of sharing them, most of these end as direct composer dependencies 

## Teachability, Documentation, Adoption, Migration Strategy
This should all be picked up from the composer.json and / or enviroment variables so shouldnt need any end use involvement

**Comment by icanhazstring** [comment]
Hi @exussum12. Thanks for reaching out.
This will be indeed a good addition 👍. Will add this in a future release.
